<?xml version="1.0"?>
<launch>

  <arg name="model" value="$(find festo_phand_description)/urdf/full_hand.xacro" />
  <arg name="gui" default="true" />
  <arg name="rvizconfig" default="$(find festo_phand_description)/rviz/urdf.rviz" />

  <param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />
  <param name="use_gui" value="$(arg gui)"/>

  <node name="joint_state_publisher" pkg="festo_phand_description" type="joint_publischer.py" />
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />

</launch>